server/rpm/pbench-server.spec

```diff
@@ -166,6 +166,9 @@ fi
 /%{installdir}/%{static}/VERSION
 /%{installdir}/%{static}/package.json
 /%{installdir}/package.json
+/%{installdir}/requirements.txt
```

Comment:
Since we only need these files during the `%post` phase, can we remove then at the end of the `%post` phase and then not include them here?

Reply:
Not sure - I think they are installed even if the post phase removes them, so it might complain about files installed but not enumerated (or whatever the complaint is). I'll try it out and see.